### ISSUE TITLE
Nan loss error

### ISSUE THREAD

**Issue by xiaoying-xing** [itaigat/removing-bias-in-multi-modal-classifiers#6]
Hi, thanks for your interesting work. I am trying to train the model but got ValueError("Nan loss"). How can I fix the problem?

**Comment by itaigat** [comment]
Hey, It is hard to tell as I don't know how you use the regularization term

**Comment by xiaoying-xing** [comment]
I tried to follow the code on Readme but it did not work. Then I checked the issues and found there used to be a training code(deleted now). Which one should I follow?

**Comment by xiaoying-xing** [comment]
And the link to the pretrained model is also unavailable. Can you please provide another one?
